Vpn.Proto/vpn.proto

```diff
@@ -17,75 +17,75 @@ package vpn;
 // msg_id which it sets on the request, the responder sets response_to that msg_id on the response
 // message
 message RPC {
```

Comment:
You might need to install editorconfig plugin for your editor: https://github.com/coder/coder-desktop-windows/blob/e0a11dd7c99d8774cc7eeedb13e62ef924879b0c/.editorconfig#L79-L81

Reply:
coder/coder version uses tabs. We need to decide on one or another and stick to it.

Reply:
Please update .editorconfig to match coder/coder in that case